client/devpi/use.py

```diff
@@ -343,6 +343,18 @@ def getvenv():
         return None
     return pip.dirpath().dirpath()
 
+def current_venv():
```

Comment:
Yes I had thought of that use case earlier on and wondered about it, forgot to take it into consideration here. I think you're right, it's probably better to take the env var as more important than the current python runtime.